recipes/newrelic/apm/node/linux.yml

```diff
@@ -24,17 +24,25 @@ preInstall:
       if [ -z "$ASSERT_NODE_INSTALLED" ] ; then
         exit 1
       fi
+      LOWEST_SUPPORTED_MAJOR_NODE_VERSION=16
+      INSTALLED_MAJOR_NODE_VERSION=$(node -v | awk -F '[^0-9]+' '{ print $2 }')
```

Comment:
It's very common to use `nvm` to manage multiple node versions, so the system node version might not be the node version that is being used to run the app. We could parse the node version from the output of `pm2 list`, similar to what's being done here:
https://github.com/newrelic/open-install-library/pull/1010/files#diff-465f7b5cc2681ce268fd0072c1926be849dedca67a68ba4a5b5776cf9dffd688L111

Reply:
Agreed on checking `nvm`, good call - though could we get by with simply performing `which` to determine then nvm directly?  Wouldn't want to install the introspector if they're not supported

Reply:
I'm making sure this works by getting the path to the binary via `command -v node`. Just confirmed it works on the EC2 instance I'm testing with and it uses `nvm` to manage node versions.

`NODEJS_BINARY_PATH=$(command -v node)` results in the following output:

```
$ command -v node
/home/ec2-user/.nvm/versions/node/v14.16.1/bin/node
```

Confirmed this works in the recipe as well.